helm-gtags.el

```diff
@@ -501,6 +501,48 @@ Always update if value of this variable is nil."
     (setq helm-gtags--last-input input)
     (reverse (cons input options))))
 
+(defun helm-gtgs--exec-global-command-local (type input)
+  (with-temp-buffer
+    (let ((args (reverse (cons input (helm-gtags--construct-options type nil)))))
+      (progn
+        (push "-a" args)
+        (apply 'process-file "global" nil '(t nil) nil args)
+        (buffer-substring-no-properties (point-min) (point-max))))))
+
+(defun helm-gtags--which-function-only-func-name ()
+  (if (string-match "lisp-mode$" (symbol-name major-mode))
```

Comment:
This function is required in order to extract function name which along with class name in C++; e.g. ClassName::function_name. global command only stores function_name part and search is capable of function_name part, too. I mean global cannot accept "ClassName::function_name" tag for search, but it accepts "function_name" tag.

I separated lisp because minus sign (-) may be included within a function name.